lib/util/sdp/simulcast.js

```diff
@@ -64,7 +64,14 @@ class TrackAttributes {
     if (this.isSimulcastEnabled) {
       return;
     }
-    const simulcastSSRCs = [createSSRC(), createSSRC()];
+
+    const originalSSRCs = Array.from(this.primarySSRCs.values());
+    let previousSSRC = 0;
+    if (originalSSRCs.length) {
+      // we will use original ssrc value to generate new values.
+      previousSSRC = Number(originalSSRCs[originalSSRCs.length - 1]);
+    }
+    const simulcastSSRCs = previousSSRC ? [String(previousSSRC + 1), String(previousSSRC + 2)] : [createSSRC(), createSSRC()];
```

Comment:
Can you explain this change?

Reply:
Here instead of generating 3 random ssrcs, we generate one random ssrc and two others in sequence. I found this to be very useful to identify the layer indexes from the ssrc numbers. We are not making any assumption in the code (or tests) on this, but I think its useful.

Reply:
now reverted